src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlNotificationTest/SqlNotificationTest.cs

```diff
@@ -174,7 +174,8 @@ public void Test_SingleDependency_AllDefaults_SqlAuth()
 
                 updateCompleted.Set();
 
-                Assert.True(notificationReceived.WaitOne(CALLBACK_TIMEOUT, false), "Notification not received within the timeout period");
+                Assert.True(notificationReceived.Wait(CALLBACK_TIMEOUT), "Notification not received within the timeout period");
+                notificationReceived.Reset();
```

Comment:
Why we reset the `notificationReceived`  to nonsignaled?

Reply:
Interesting find.
I don't see any reason to make this change, taking out.